modules/ossm-release-2-4-11.adoc

```diff
@@ -7,7 +7,7 @@ Module included in the following assemblies:
 [id="ossm-release-2-4-11_{context}"]
 = {SMProductName} version 2.4.11
 
-This release of {SMProductName} includes 2.6.2, 2.5.5, and 2.4.11; addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {product-title} 4.14 and later.
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and is supported on {product-title} 4.14 and later. This version is included in {SMProductName} Operator 2.6.2.
```

Comment:
Reverse the phrasing: This release of {SMProductName} is included with the {SMProductName} Operator 2.6.2, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {product-title} 4.14 and later.